grumphp.yml

```diff
@@ -38,7 +36,7 @@ grumphp:
                 no_exit_statements: ~
                 never_use_else: ~
                 forbidden_function_calls:
-                    blacklist: [var_dump, echo, print, dump, print_r, phpinfo]
```

Comment:
Should we still be using the phpparser task?
You can also specify a blacklist through psalm
https://github.com/vimeo/psalm/blob/c97ba8f7135aa23cfe5800f216c9a11c4d7e2c29/tests/Config/ConfigTest.php#L1006-L1008

I am pretty sure phpmd and phpstan have similar options.


To be honest : I am even considering removing the phpparser task from grumphp, because static analysis evolved this much. 

For the never_use_else, there most likely also is a psalm plugin (or you could create on)
It surely is inside phpmd and there is also an advanced phpcs config for object calisthenics inside https://github.com/slevomat/coding-standard


So the tools that phpparser provide are in my opinion not really needed anymore.